src/NetMQ/Core/Patterns/Server.cs

```diff
@@ -70,7 +67,7 @@ public Server([NotNull] Ctx parent, int threadId, int socketId)
         /// <param name="icanhasall">not used</param>
         protected override void XAttachPipe(Pipe pipe, bool icanhasall)
         {
-            Debug.Assert(pipe != null);
+            Assumes.NotNull(pipe);
```

Comment:
I don't think this is needed anymore, not that SocketBase enabled nullable, probably for all sockets

Reply:
Will remove.